### PR TITLE
Generate global functions

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/295
Apply https://github.com/gtk-rs/gir/pull/371

### CODE REVIEW

File: src/auto/functions.rs [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+use Stretch;
+use Style;
+use Variant;
+use Weight;
+use ffi;
+use glib::translate::*;
+use std::mem;
+
+
+//#[cfg(feature = "v1_38")]
```
Comment (GuillaumeGomez) [inline src/auto/functions.rs]: That's a lot of commented functions. :p

**Comment by GuillaumeGomez** [comment]
Thanks!

**Comment by sdroege** [comment]
Do I see that right that not a single of those functions is actually exported? :)

**Comment by EPashkin** [comment]
Yes, I say it from start ;)

**Pull request merged**